source/components/SettingsMenu.js

```diff
@@ -0,0 +1,175 @@
+                    </svg>          
+                    <p>SFX</p>
+                    <input type="range" min="0" max="100" value="50" class="slider" id="sfxRange">
+                </div>
+                <div class="slidercontainer">
+                    <svg width="30" height="30" viewBox="0 0 58 89" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M19.8748 30.75L57.0623 17.5M57.0623 46.4424V63.0049C57.0619 65.1642 56.4976 67.2646 55.4547 68.9882C54.4119 70.7119 52.9472 71.9651 51.2823 72.5582L46.6073 74.2232C45.7967 74.5287 44.9454 74.6301 44.1031 74.5214C43.2608 74.4128 42.4443 74.0963 41.701 73.5904C40.9577 73.0845 40.3025 72.3993 39.7735 71.5746C39.2446 70.75 38.8525 69.8024 38.62 68.7869C38.3874 67.7715 38.3192 66.7086 38.4192 65.6599C38.5192 64.6113 38.7855 63.598 39.2025 62.6789C39.6195 61.7598 40.1789 60.9533 40.8482 60.3064C41.5175 59.6595 42.2833 59.185 43.101 58.9107L51.2823 55.9957C52.9472 55.4026 54.4119 54.1494 55.4547 52.4257C56.4976 50.7021 57.0619 48.6017 57.0623 46.4424ZM57.0623 46.4424V0.9375L19.8748 14.1875V59.6924M19.8748 59.6924V76.2549C19.8744 78.4142 19.3101 80.5146 18.2672 82.2382C17.2244 83.9619 15.7597 85.2151 14.0948 85.8082L9.41977 87.4732C7.80071 88.0165 6.07527 87.7439 4.61602 86.7142C3.15677 85.6846 2.0808 83.9805 1.62044 81.97C1.16007 79.9595 1.35226 77.8038 2.15549 75.9685C2.95872 74.1332 4.30855 72.7655 5.91352 72.1607L14.0948 69.2457C15.7597 68.6526 17.2244 67.3994 18.2672 65.6757C19.3101 63.9521 19.8744 61.8517 19.8748 59.6924Z" stroke="black" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+                    </svg>
+
+                    <p>Music</p>
+                    <input type="range" min="0" max="100" value="50" class="slider" id="musicRange">
+                </div>
+
+                <h1>Language</h1>
+                <select id="language">
+                    <option value="en">English</option>
+                    <option value="es">Spanish</option>
+                    <option value="fr">French</option>
+                </select>
+            </div>
+            <audio id="audioMusic" class="hidden" src="./audio/music_hedwigs_theme.mp3" loop></audio>
+            <audio id="audioSfx" class="hidden" src="./audio/sfx_harry_potter_expecto_patronum_sound_effect.mp3"></audio>
+
+        `;
+
+    this.shadowRoot.adoptedStyleSheets = [normalize, style];
+
+    document
+      .querySelector('app-header')
+      .shadowRoot.querySelector('#settings > svg')
+      .addEventListener('click', () => {
```

Comment:
we can leave this for now, or we can emit a custom event in `AppHeader` and this component will listen on that event